ACE/ace/os_include/os_pthread.h

```diff
@@ -45,9 +45,13 @@
 
 
 #if !defined (ACE_LACKS_PTHREAD_H)
+# if !defined ACE_VXWORKS || ACE_VXWORKS < 0x700
```

Comment:
Would there really be a platform that needs this? Probably this was in the ancient ACE_OS code and nobody tried to remove it until now